### PR TITLE
Improve dashboard a11y

### LINKED ISSUE #13485: Help! Links must have discernible text
### Summary
The 2.x default theme utilizes icon fonts in an inaccessible manor:

```html
<a href="?a=help" title=""><i class="icon-question-circle icon icon-large"></i></a>
```

It would be great to doing a sweep and fix this every where in the manager, but I think it is important for at least the Help! button to be accessible.

### Step to reproduce
Open the Manager

### Observed behavior
Can't access the help link using assistive technology

### Expected behavior
Should be able to access the help link using assistive technology

### Environment
MODX 2.x

### Solution

```html
<a href="?a=help" title="" aria-label="Help!"><i class="icon-question-circle icon icon-large"></i></a>
```

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
Improving accessibility in the control panel

### Why is it needed?

- Fix `target="_blank"` links: a security and performance issue in web page
- Fixing the meta tag charset
- Added an attribute aria-label for buttons widgets on the home page
- Correcting color contrast in Who's Online and Recently Edited Resources panels (WCAG 2 AA)
- Added aria-label parameter for the left menu

### Related issue(s)/PR(s)
Resolves #13485

### CODE REVIEW

**Review by theboxer — APPROVED** [review]

File: manager/controllers/default/header.php [hunk 1]
```diff
@@ -188,11 +188,11 @@ public function buildMenu($name, $placeholder)
                     $menu['action'] .= '&namespace='.$menu['namespace'];
                 }
                 $onclick = (!empty($menu['handler'])) ? ' onclick="'.str_replace('"','\'',$menu['handler']).'"' : '';
-                $menuTpl .= '<a href="?a='.$menu['action'].$menu['params'].'"'.( $top ? ' class="top-link"': '' ).$onclick.$title.'>'.$label.$description.'</a>'."\n";
+                $menuTpl .= '<a href="?a='.$menu['action'].$menu['params'].'"'.' aria-label="'.$menu['text'].'" '.( $top ? ' class="top-link"': '' ).$onclick.$title.'>'.$label.$description.'</a>'."\n";
```
Comment (Mark-H) [inline manager/controllers/default/header.php:191]: Is $menu['text'] already encoded somewhere to avoid XSS insertion in the menu text?

Comment (Ibochkarev) [reply manager/controllers/default/header.php:191]: @Mark-H I think not - how can I fix it?

Comment (Mark-H) [reply manager/controllers/default/header.php:191]: Just wrapping the variable with htmlentities does the trick.

**Review by theboxer — CHANGES_REQUESTED** [review]
@Ibochkarev will you be able to resolve Mark's request?

File: manager/controllers/default/header.php (hunk 1 above)
Comment (rthrash) [reply manager/controllers/default/header.php:191]: @Mark-H would the following work for @Ibochkarev to address the XSS concern (also adding to lines 193, 195 and 328, below)? 

`' aria-label="'.htmlentities($menu['text'], ENT_QUOTES, 'UTF-8').'" '.`

Comment (Mark-H) [reply manager/controllers/default/header.php:191]: Yeah that's how that would normally be handled. It does look like it's also not being applied to the $title and the $description so this whole thing might be a problem. Or, perhaps it's already handled elsewhere, I haven't gone through the getCache() method reponsible for fetching menus to see if that's sanitising it.

**Pull request closed without merging**